core/replay/connection_thread.py

```diff
@@ -206,28 +206,6 @@ def execute_transactions(self, connection):
             for transaction in self.connection_log.transactions:
                 self.execute_transaction(transaction, connection)
 
-    def save_query_stats(self, start_time, end_time, xid, query_idx):
```

Comment:
Is this not called from anywhere?

Reply:
It is, I removed the bit that calls this method. The csvs are unused in a sense that they are generated, but not used at a later point. This method is used to generates the csv files during replay.